baram/efs_manager.py

```diff
@@ -0,0 +1,79 @@
+import boto3
+import traceback
+
+from baram.log_manager import LogManager
+
+
+class EFSManager(object):
+    def __init__(self):
+        self.cli = boto3.client('efs')
+        self.logger = LogManager.get_logger()
+
+    def list_efss(self):
+        """
+        List one or more of your EFS.
+
+        :return:
+        """
+        return self.cli.describe_file_systems()['FileSystems']
+
+    def list_mount_targets(self, efs_id: str):
+        """
+        List mount targets of specific file system.
+
+        :param efs_id: FileSystemId
+        :return:
+        """
+        try:
+            return self.cli.describe_mount_targets(FileSystemId=efs_id)['MountTargets']
```

Comment:
method should not make exception with empty result.

Reply:
add `return None` instead.